syn/python/get_kge.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Parse a yosys area report and give a kGE equivalient
+
+import argparse
+
+
+def read_lib(lib_file_path):
+    with open(lib_file_path, 'r') as f:
+        lib_file = f.readlines()
+
+    cell_dict = {}
+    weighted_dict = {}
+    cell_name = ''
```

Comment:
I'd consider using `None` instead of the empty string here. Or, I suppose, you could assert that `cell_name` isn't empty after line 24.

Reply:
Will do